.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.0.0
+2.0.0
```

Comment:
I'd changed the version number while editing the gem because my ruby package manager(rbenv) didn't have ruby-2.0.0 and I made a mistake changing it back. 

I think this is an area I'd like someone who's more familiar with gems than myself to comment on. Should we change this ruby version to just be greater than ruby 2.0.0?

Reply:
Actually this is not where we define the ruby version needed. Instead I think this is just for rbenv. We can update this to match the api gemfile and just require greater than 2.0.0